libs/image/CMakeLists.txt

```diff
@@ -40,3 +43,11 @@ target_compile_options(${TARGET} PRIVATE
 # ==================================================================================================
 add_library(image_headers INTERFACE)
 target_include_directories(image_headers INTERFACE ${PUBLIC_HDR_DIR})
+
+# ==================================================================================================
+# Tests
+# ==================================================================================================
+add_executable(test_${TARGET}
```

Comment:
You need to compile this only on host targets, not Android.

Reply:
Done